torch/nn/utils/fusion.py

```diff
@@ -0,0 +1,29 @@
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import copy
+import torch
+
+def fuse_conv_bn_eval(conv, bn):
+    assert(not (conv.training or bn.training)), "Fusion only for eval!"
+    fused_conv = copy.deepcopy(conv)
+
+    w_conv = fused_conv.weight
+    b_conv = fused_conv.bias
+
+    bn_mean = bn.running_mean
+    bn_var_sqrt = torch.sqrt(bn.running_var + bn.eps)
+
+    bn_weight = bn.weight
+    bn_bias = bn.bias
+
+    if b_conv is None:
+        b_conv = bn_mean.new_zeros(bn_mean.shape)
+
+    w_conv = w_conv * (bn_weight / bn_var_sqrt).reshape([-1, 1, 1, 1])
```

Comment:
two divisions by bn_var_sqrt could be replaced by multiplication if rsqrt is used instead of sqrt (not sure if it matters much)

Reply:
Will send optimization as a separate diff to unblock

Reply:
#23275